cost/azure/schedule_instance/azure_schedule_instance.pt

```diff
@@ -146,7 +146,7 @@ datasource "ds_azure_instances" do
   end
   result do
     encoding "json"
-    field "status", jq(response,'.statuses[1].code/"/"|.[1]')
+    field "status", jq(response,'.statuses[] | select( .code | match("PowerState")) | .code[11:50]')
```

Comment:
What is `| .code[11:50]` used for?  Can we add a comment breaking down the logic here.  Without knowing anything about the datasource or API I wouldn't expect anything more than `selecct( .code | match("PowerState"))`

Reply:
updated with split recommendation